tests/test_kernels.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+from _pytest.python_api import approx
+
+from pywhy_stats.kernels import estimate_squared_sigma_rbf
+
+
+def test_given_simple_data_when_estimate_squared_sigma_rbf_then_returns_correct_results():
+    X = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+
+    assert estimate_squared_sigma_rbf(X, method='silverman') == approx(0.3758902254410145)
```

Comment:
Where did these values come from? Can we document it in the in-line comment, or docstring of the test?